File: Algorithms/1029. Two City Scheduling/javascript/solution.js
/**
 * @param {number[][]} costs
 * @return {number}
 */
var twoCitySchedCost = function (costs) {
    costs.sort((a, b) => a[0] - a[1] - (b[0] - b[1]));
    let total = 0;
    const n = costs.length / 2;
    for (let i = 0; i < n; i++) {
        total += costs[i][0] + costs[i + n][1];
    }
    return total;
};

File: Algorithms/1029. Two City Scheduling/golang/solution.go
func twoCitySchedCost(costs [][]int) int {
	sort.Slice(costs, func(i, j int) bool {
		return costs[i][0]-costs[i][1] < costs[j][0]-costs[j][1]
	})
	var total int
	n := len(costs) / 2
	for i := 0; i < n; i++ {
		total += costs[i][0] + costs[i+n][1]
	}
	return total
}
